src/Microsoft.AspNet.SignalR.Crank/PerformanceCounters.cs

```diff
@@ -88,9 +88,9 @@ private static PerformanceCounter LoadCounter(string category, string name, stri
                 counter.NextSample();
                 return counter;
             }
-            catch (Exception)
+            catch (Exception x)
             {
-                Console.WriteLine("Unable to load counter '{0}\\{1}' on host '{2}'", category, name, host);
```

Comment:
My performance counters were loading, which I think is a permissions issue, however i wasnt being told that. this will now give a bit more info